src/current/v24.3/wal-failover.md

```diff
@@ -0,0 +1,495 @@
+
+<a name="3-sidecar-configure-logs"></a>
+
+#### 3. Log Configuration for WAL Failover
+
+If you are logging to a file-based sink, create a `logs.yaml` file. Later on, you will pass this file to `cockroach start` when starting each node in the cluster. Not doing so will negate the positive impact of enabling WAL failover, because writes to the diagnostic logs may block indefinitely during the disk stall, effectively stalling the node.
+
+{% include {{ page.version.version }}/wal-failover-log-config.md %}
+
+Next transfer this file to the nodes on the cluster.
+
+For more information about how to configure CockroachDB logs, see [Configure logs]({% link {{ page.version.version }}/configure-logs.md %}).
+
+<a name="4-sidecar-prepare-failover-disk"></a>
+
+#### 4. Preparing the WAL Failover side disk on the cluster
+
+To enable WAL failover on a single data store cluster with a side disk, you need to:
+
+1. Make sure that each node has an additional disk available for WAL failover before starting the cluster (as described in [Step 1](#1-sidecar-create)).
+2. Know the path to this side disk.
+
+To find out the path to side disk, `ssh` to one of the nodes in the cluster and use the `lsblk` command to see the path to the `data2` volume. The output should look like the following:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+NAME    MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+loop0     7:0    0  63.9M  1 loop /snap/core20/2182
+loop1     7:1    0 354.3M  1 loop /snap/google-cloud-cli/223
+loop2     7:2    0    87M  1 loop /snap/lxd/27428
+loop3     7:3    0  39.1M  1 loop /snap/snapd/21184
+sda       8:0    0    10G  0 disk
+├─sda1    8:1    0   9.9G  0 part /
+├─sda14   8:14   0     4M  0 part
+└─sda15   8:15   0   106M  0 part /boot/efi
+nvme0n1 259:0    0   375G  0 disk /mnt/data1
+nvme0n2 259:1    0   375G  0 disk /mnt/data2
+~~~
+
+Here we can see the path to `data2` volume is `/mnt/data2`. For each node, create the directory `/mnt/data2/cockroach` on the side disk volume where the WAL failover data can be written. `/mnt/data1` will be used as the single user data store for the cluster.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+mkdir /mnt/data2/cockroach
+~~~
+
+<a name="5-sidecar-start"></a>
+
+#### 5. Starting the cluster with WAL failover enabled
+
+To enable WAL failover when we start the cluster,  we can either pass the [`--wal-failover=path={ path-to-my-side-disk-for-wal-failover }` to `cockroach start`]({% link {{ page.version.version }}/cockroach-start.md%}#enable-wal-failover)
+
+ *OR*
+
+set [the environment variable `COCKROACH_WAL_FAILOVER=path={ path-to-my-side-disk-for-wal-failover }`]({% link {{ page.version.version }}/cockroach-start.md%}#enable-wal-failover) before starting the cluster.
+
+Additionally, we must set the value of the environment variable `COCKROACH_ENGINE_MAX_SYNC_DURATION_DEFAULT` to `40s`. This variable controls the duration of stall when CockroachDB crashes the node.
```

Comment:
I feel like this might benefit from more context. Maybe something like:

> Additionally, we must set the value of the environment variable `COCKROACH_ENGINE_MAX_SYNC_DURATION_DEFAULT` to `40s`. By default, CockroachDB detects prolonged stalls and crashes the node after 20s. With WAL failover enabled, CockroachDB should be able to survive stalls of up to 40s with minimal impact to the workload

Reply:
thanks i have updated to use that language here and in the other section where we discuss setting this env var